client/my-sites/plans/current-plan/jetpack-product-install/index.js

```diff
@@ -0,0 +1,239 @@
+/**
+ * External dependencies
+ */
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { some } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import getJetpackProductInstallProgress from 'state/selectors/get-jetpack-product-install-progress';
+import getJetpackProductInstallStatus from 'state/selectors/get-jetpack-product-install-status';
+import Interval, { EVERY_SECOND, EVERY_FIVE_SECONDS } from 'lib/interval';
+import Notice from 'components/notice';
+import NoticeAction from 'components/notice/notice-action';
+import QueryPluginKeys from 'components/data/query-plugin-keys';
+import { JETPACK_CONTACT_SUPPORT } from 'lib/url/support';
+import { getPluginKeys } from 'state/plugins/premium/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import {
+	requestJetpackProductInstallStatus,
+	startJetpackProductInstall,
+} from 'state/jetpack-product-install/actions';
+
+/**
+ * Module variables
+ */
+/**
+ * These are plugin states in the installation lifecycle we consider "non-error" states.
+ */
+const NON_ERROR_STATES = [
+	'not_active', // Plugin is not installed
+	'option_name_not_in_whitelist', // Plugin is installed but not activated
+	'key_not_set', // Plugin is installed and activated, but not configured
+	'installed', // Plugin is installed, activated and configured
+];
+/**
+ * Those errors are any of the following:
+ * - Temporary, occuring if we request installation status while plugin is being set up.
```

Comment:
`occuring` -> `occurring`

Reply:
👍 thanks, fixed in https://github.com/Automattic/wp-calypso/pull/31684/commits/3f482a53d31b56e4837dc1852d6945e6d4f8e609

Reply:
Thank you!